src/inverseRobot.js

```diff
@@ -7,7 +7,25 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+  const propertyValues = Object.values(robot);
+  let current;
+
+  for (let i = 0; i < propertyValues.length; i++) {
+    current = propertyValues[i];
+
+    for (let j = i + 1; j < propertyValues.length; j++) {
+      if (current === propertyValues[j]) {
+        return null;
+      }
+    }
```

Comment:
The current implementation checks for repeated values by comparing each value with all subsequent values. This can be optimized by using a Set to keep track of unique values encountered.